app/src/main/java/com/alexbar/layoutbasic/screen/confirm_payment/widgets/ShowComponent.kt

```diff
@@ -0,0 +1,97 @@
+package com.alexbar.layoutbasic.screen.confirm_payment.widgets
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.ElevatedCard
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import com.alexbar.layoutbasic.createVerticalGradient
+import com.alexbar.layoutbasic.ui.theme.Dimens.dimen_12_dp
+import com.alexbar.layoutbasic.ui.theme.Dimens.dimen_180_dp
+import com.alexbar.layoutbasic.ui.theme.Dimens.dimen_250_dp
+import com.alexbar.layoutbasic.ui.theme.Dimens.dimen_8_dp
+import com.alexbar.layoutbasic.utils.Media
+import com.alexbar.layoutbasic.utils.Movie
+import com.alexbar.layoutbasic.utils.ShowsConstants.base_url
+import com.alexbar.layoutbasic.utils.TVSeries
+
+@Composable
+fun MediaComponent(
```

Comment:
Your code looks pretty clean and well-organized. However, here you need to have some changes based on the changes on your data class